components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -81,6 +83,7 @@
 #include "third_party/dom_distiller_js/dom_distiller.pb.h"
 #include "ui/base/l10n/l10n_util.h"
 #include "ui/base/resource/resource_bundle.h"
+#include "ui/message_center/public/cpp/notification.h"
 
 #if defined(OS_ANDROID)
 #include "chrome/browser/android/service_tab_launcher.h"
```

Comment:
@yachtcaptain23 Is `#include "brave/browser/notifications/brave_notification_platform_bridge_helper_android.h"` not needed here, as it was included before we moved to custom ad notifications?

Reply:
Will follow up

Reply:
I'll be testing and posting the results, but the code defines `MaybeRegenerateNotification` which we don't use.

Reply:
Adding back as required to limit amount of ad notifications in the system tray on Android